File: XBMCSkinEditor/Controls/ErrorLog.vb
Imports System.IO
Imports WeifenLuo.WinFormsUI

Public Class ErrorLogWindow
    Inherits DockContent

    Public Sub ErrLog(ByVal errortxt As String)
        Me.lstErrors.Items.Add(errortxt)
    End Sub

    'Public Property ErrorItems()
    '    Get
    '        Return strID
    '    End Get
    '    Set(ByVal Value As String)
    '        strID = Value
    '    End Set
    'End Property
End Class

File: XBMCSkinEditor/Components/ErrorLog.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using WeifenLuo.WinFormsUI.Docking;
namespace XBMCSkinEditor.Components
{
    public partial class ErrorLog : DockContent
    {
        public ErrorLog()
        {
            InitializeComponent();
        }

        public void AddError(string errortxt) 
        {
            this.lstErrors.Items.Add(errortxt);
        }
        public void ClearErrors(string errortxt) 
        {
            this.lstErrors.Items.Clear();
        }

        private void ErrorLog_Load(object sender, EventArgs e)
        {

        }

    }
}
